Sources/TensorFlow/BackwardsCompatibility.swift

```diff
@@ -14,6 +14,137 @@
 
 // TODO: Remove this file after 0.4.
 
+public extension Tensor where Scalar == Int32 {
+    /// Creates a tensor with the specified shape, randomly sampling scalar values from a discrete 
+    /// uniform distribution.
+    ///
+    /// - Parameters:
+    ///   - shape: The dimensions of the tensor.
+    ///   - generator: Random number generator to use.
+    ///   - lowerBound: The lower bound of the distribution.
+    ///   - upperBound: The upper bound of the distribution.
+    @available(*, deprecated, message: "This API will be removed after Swift for TensorFlow 0.4.")
+    init<G: RandomNumberGenerator>(
+        randomUniform shape: TensorShape,
+        generator: inout G,
+        lowerBound: Scalar = Scalar.self.min,
```

Comment:
No need to use metatype here.
```suggestion
        lowerBound: Scalar = Scalar.min,
```